bot/actions/action_product_list.py

```diff
@@ -25,21 +33,31 @@ def run(
         # TODO: query VTEX's product list database to match the products
         # with what the user has ordered
 
-        vtex_store = os.getenv('VTEX_STORE')
-        vtex_store_api_key = os.environ.get('VTEX_STORE_API_KEY')
-        vtex_store_api_token = os.environ.get('VTEX_STORE_API_TOKEN')
+        # vtex_store = os.getenv('VTEX_STORE')
+        # vtex_store_api_key = os.environ.get('VTEX_STORE_API_KEY')
+        # vtex_store_api_token = os.environ.get('VTEX_STORE_API_TOKEN')
+
+        entities = tracker.latest_message["entities"]
+        logger.info(f"Extracted entities: {pformat(entities)}")
+
+        message = "Lista de produtos: "
+        current_group = 1
+
+        for current_group in range(1, len(entities)):
```

Comment:
My logic here so far is that we'll have at most `len(entities)` groups (if no quantity is given for each, for example), so I'm trying to grab what data we may have for the max amount of entity groups we have.